datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/MutableType.java

```diff
@@ -15,6 +17,10 @@
      */
     Class<I> inputClass();
 
+    default Class<I[]> arrayInputClass() throws UnsupportedOperationException {
```

Comment:
alternative that is being suggested above: consider changing this to batchInputClass() and just having it be a single class!